grammar.js

```diff
@@ -2273,13 +2275,31 @@ module.exports = grammar({
             seq(
               $.keyword_on,
               $.keyword_delete,
-              $.keyword_cascade,
+              $._referential_action,
```

Comment:
this could be combined in one optional block with a choice between `keyword_delete` and `keyword_update`. I see no difference between the other parts of the sequence.